src/components/search/search.stories.mdx

```diff
@@ -177,6 +177,101 @@ In this example the `variant` prop is "dark" and no search button is present.
   </Story>
 </Canvas>
 
+### Validations
+
+Validation status can be set by passing `error`, `warning` or `info` prop to the component
+
+Passing a string to these props will display a properly colored border along with a validation icon and tooltip - string value will be displayed as the tooltip message.
+
+Passing a boolean to these props will display only a properly colored border.
+
+For more information check our [Validations](?path=/docs/documentation-validations--page "Validations") documentation page
+
+#### As a string
+
+<Canvas>
+  <Story name="validations - string - component">
+    {() => {
+      const [state, setState] = useState({
+        error: "<>/",
```

Comment:
I know you want to show something with "invalid characters" to make it look realistic, but this looks weird to me with those values. I'd put some plain text in there, maybe with one invalid character like `foo/`? Up to you, I don't mind leaving this as it is if you prefer it.